config/routes.rb

```diff
@@ -64,6 +64,9 @@
 
     patch "/orders/:order_id/items/:item_id/update", to: "order_items#update"
 
+    get "/discounts/new", to: "discounts#new"
```

Comment:
Very tricky that the order of these has to be above the index page. Error message thrown is NOT helpful